src/main/java/racingcar/domain/RacingCarGame.java

```diff
@@ -0,0 +1,64 @@
+package racingcar.domain;
+
+import java.util.Objects;
+import java.util.regex.Pattern;
+import racingcar.ErrorMessage;
+
+public class RacingCarGame {
+
+    private static final int MIN_ROUND = 1;
+
+    private static final String NUMBER_PATTERN = "^[0-9]*$";
```

Comment:
패턴 객체의 생성 비용이 크기 때문에 다음처럼 패턴을 생성해놓고 재사용하는 편입니다 :)
```suggestion
    private static final Pattern NUMBER_PATTERN = Pattern.compile("^[0-9]*$");
```